api/v1alpha1/selfnoderemediationconfig_webhook_test.go

```diff
@@ -77,6 +94,33 @@ func testInvalidCRs(validationType string) {
 	}
 }
 
+func testMultipleInvalidFields(validationType string) {
+	var errorMsg string
+	snrc := createDefaultSelfNodeRemediationConfigCR()
+
+	for _, item := range testItems2 {
+		item := item
+		setFieldValue(snrc, item.name, item.durationValue)
+		errorMsg += "\n" + item.name + " cannot be less than " + item.minDurationValue.String()
+	}
+	println(errorMsg)
```

Comment:
Is that necessary? I assume it was merely for debugging.

Reply:
good catch! thanks